src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java

```diff
@@ -121,29 +119,22 @@ public FSFileInfo copyFile(Dataset dataset,
             // From poking around I think it is a standard POSIX milliseconds since Jan 1, 1970.
             Instant createTime = Instant.ofEpochMilli(targetBlob.getCreateTime());
 
-            URI gspath = new URI("gs",
-                bucketResource.getName(),
-                "/" + targetPath,
-                null,
-                null);
+            String gspath = String.format("gs://%s/%s", bucketResource.getName(), targetPath);
```

Comment:
Do we have any security concerns generating these `gspath`s?

Reply:
I tried to thing of some but I can't think of them since we create these paths based on valid source GCS paths and we can't use `URI.create` `gs://` URIs appear to support values that aren't compatible with it (e.g. `gs://mybucket/file#.txt`)